cg/apps/cgstats/stats.py

```diff
@@ -145,6 +145,8 @@ def fastqs(self, flowcell: str, sample_obj: Sample) -> Iterator[Path]:
         """Fetch FASTQ files for a sample."""
         base_pattern = "*{}/Unaligned*/Project_*/Sample_{}/*.fastq.gz"
         alt_pattern = "*{}/Unaligned*/Project_*/Sample_{}_*/*.fastq.gz"
-        for fastq_pattern in (base_pattern, alt_pattern):
+        bcl_convert_pattern = "*{}/Unaligned*/*/Sample_{}_*/*.fastq.gz"

```

Comment:
Should we have a pattern for matching fastq files without a trailing sequence in the new structure as well?

Reply:
I don't have knowledge to answer that

Reply:
I guess this would make sense if we encounter such patterns for fastq files. Maybe we can refine the issue given a bit further on what the expected fastq files can be currently. Then we can tailer our patterns specifically to that.

Reply:
I think @karlnyr has the answer 🙏

Reply:
Not relevant based on the example file path provided in the issue